cs-amazon/pom.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
```

Comment:
@pintea-eugen  @tethryus I understand we need the plugin, I don't understand when was the parent tag introduced in the discussion... Don't misunderstand me, I agree with the parent tag :)